README.md

```diff
@@ -1,6 +1,7 @@
 # Project Oak
 
 [![Build Status](https://travis-ci.org/project-oak/oak.svg?branch=master)](https://travis-ci.org/project-oak/oak)
+[![Fuzzing Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/oak.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:oak)
```

Comment:
The page that links to doesn't show any results for `proj:oak` for me (even when I sign in) &ndash; is that expected?

(If the link leads to a page that has access controls then I'd lean towards not having it on a public repo, btw)

Reply:
Also, I think the page *will* show issues once they are disclosed (90 days by default).